tensorboard/plugins/text/summary.py

```diff
@@ -92,7 +92,7 @@ def pb(name, data, display_name=None, description=None):
   Returns:
     A `tf.Summary` protobuf object.
   """
-  if isinstance(data, six.binary_type) or isinstance(data, six.string_types):
+  if isinstance(data, six.binary_type, six.string_types):
```

Comment:
`isinstance(data, (six.binary_type, six.string_types))`

`python -c 'print isinstance.__doc__'`

Reply:
Ah, thanks! Done. Also, that means of printing docs is useful to know.